control-plane-operator/controllers/hostedcontrolplane/assets/cluster-bootstrap/node-bootstrapper-csr-approval-rolebinding.yaml

```diff
@@ -0,0 +1,13 @@
+# Approve all CSRs for the group "system:bootstrappers"
```

Comment:
this clusterrolebinding will let kube-controller-manager approve kubelet client csr. reference: https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping/#approval

Reply:
can you add this comment as a comment in the file?